docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+
+volumes:
+  db-data:
+    external: false
+
+services:
+  proxy:
+    extends:
+      file: docker-compose.common.yml
+      service: proxy
+
+
+  database:
+    extends:
+      file: docker-compose.common.yml
+      service: proxy
+    volumes:
+      - db-data:/var/lib/postgresql/db-data
```

Comment:
I think your problem is you can't use `external` with `driver`, so try comment `external: false` (I also think `external: false` is the default option) and write instead `driver: local`.